main_tests/adrv9361_bob/rf_test.sh

```diff
@@ -3,6 +3,6 @@ ANSWER=1
 SCRIPT_DIR="$(readlink -f $(dirname $0))"
 
 echo
-python3 -m pytest --color yes -vs $SCRIPT_DIR/../work/pyadi-iio/test/test_fmcomms2-3_prod.py --uri="ip:analog.local" --adi-hw-map --hw=adrv9361
+python3 -m pytest --color yes -vs $SCRIPT_DIR/../work/pyadi-iio/test/test_adrv9361_prod.py --uri="ip:analog.local" --adi-hw-map --hw=adrv9361
```

Comment:
does this test_adrv9361_prod.py file actually exist?

Reply:
It was added in the fmcomms_scpi branch, commit [476bbcc](https://github.com/analogdevicesinc/pyadi-iio/commit/476bbccecd42616f6cb6217bc9cf118e18e71b42).